tests/from.bats

```diff
@@ -345,3 +346,24 @@ load helpers
   run_buildah --log-level=error containers -f id=${cid}
   buildah rm ${cid}
 }
+
+@test "from with/without pull test" {
+  run buildah from --pull=false docker.io/nginx
+  [ $status -ne 0 ]
+  run buildah from --name nginxc docker.io/nginx
+  expect_output --substring "Getting"
+  run buildah from --name nginxc-1 docker.io/nginx
+  [[ "$output" != "Getting"* ]]
```

Comment:
See above comment re: this not doing what you think.

Reply:
I test it in my fedora31 and it works as expect.
Here is the code:
        run buildah pull busybox
        echo $output
        [[ "$output" != "Getting"* ]]
#   `[[ "$output" != "Getting"* ]]' failed with status 127
# 
# Getting image source signatures Copying blob sha256:0f8c40e1270f10d085dda8ce12b7c5b17cd808f055df5a7222f54837ca0feae0 Copying config sha256:020584afccce44678ec82676db80f68d50ea5c766b6e9d9601f7b5fc86dfb96d Writing manifest to image destination Storing signatures 020584afccce44678ec82676db80f68d50ea5c766b6e9d9601f7b5fc86dfb96d

Reply:
Yes, it works, but it is not a good test: it does not check what you think it checks. You want to fail if "Getting" appears anywhere in the output; this test only fails if the output begins with "Getting". Which in most cases it will, but not if there are warning messages or any other output. Please see my comments a few minutes ago about testing string output.